### ISSUE TITLE
Weird request: unsing previous major of rxjs with Angular 14

### ISSUE THREAD

**Issue by janwidmer** [tonysamperi/ngx-popperjs#38]
**Describe the bug**
Thank for your work on this Repository. I know, this is not a bug from this repository! I have read all the hints from your side regarding creating bugs. Since it would be the easiest solution for us, I did not want to missout this option.

In our Project stack, we are forced to use RXJS Version 6.6.0 due to another library in it's current version is using RXJS Version 6.6.0. I cannot use an older version of ngx-popperjs because our whole stack uses angular 14.

I wanted to ask, if it would be possible to publish an ngx-popperjs version using RXJS 6.6.0 or is there a specific reason, which would it make unpossible to use a lower RXJS Version? I would be happy to do the code changes needed for it.

**To Reproduce**

1. Setup An angular Project with Angular 14 and RXJS 6.6.0
2. Use ngx-popperks
3. Build does not work

**Expected behavior**
ngx-popperjs uses a lower version of RXJS

**Desktop (if execution):**
 - OS: all
 - Browser all

** Versions
 - ngx-popperjs version 14.1.6
 - @angular/core version 14.2.12
 - typescript version 4.7.2

**Comment by tonysamperi** [comment]
TLDR: no. I don't adapt libraries to weird architectures people may have.

You can find a workaround on your code base, depending on your project's complexity.
But trust me that it's not impossibile.

**Comment by janwidmer** [comment]
ok, thanks for answering
